metricbeat/module/elasticsearch/elasticsearch_integration_test.go

```diff
@@ -261,10 +261,11 @@ func getConfig(metricset string, host string) map[string]interface{} {
 
 func getXPackConfig(host string) map[string]interface{} {
 	return map[string]interface{}{
-		"module":        elasticsearch.ModuleName,
-		"metricsets":    xpackMetricSets,
-		"hosts":         []string{host},
-		"xpack.enabled": true,
+		"module":                     elasticsearch.ModuleName,
+		"metricsets":                 xpackMetricSets,
+		"hosts":                      []string{host},
+		"xpack.enabled":              true,
+		"index_recovery.active_only": false,
```

Comment:
@ycombinator I had to add this line `index_recovery.active_only` so that this test passes https://github.com/elastic/beats/blob/master/metricbeat/module/elasticsearch/elasticsearch_integration_test.go#L120

The reason is that the `index_recovery` metricset has a field called `active_only` that is by default to true so if no index is being recovered (which is the case of the running container in the CI) an empty JSON `{}` is returned and the test fails miserably.

Does it makes sense?

Reply:
Yes, it makes sense. I would suggest adding an explanatory comment next to this config line so we know why it's there in the future.

Reply:
Good idea. I have added a comment. Thanks!